src/components/TimeAgo.js

```diff
@@ -0,0 +1,36 @@
+import { distanceInWords, format } from 'date-fns';
+import React from 'react';
+
+class TimeAgo extends React.Component {
+  static defaultProps = {
+    render: relativeTime => `Last updated ${relativeTime} ago.`,
+    intervalTimeout: 60000,
+  };
+
+  state = { dateToCompare: Date.now() };
+
+  componentDidMount() {
+    this.interval = setInterval(this.handleTick, this.props.intervalTimeout);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  handleTick = () => {
+    this.setState({ dateToCompare: Date.now() });
+  };
+
+  render() {
+    const { date, render } = this.props;
+    const { dateToCompare } = this.state;
+    
+    return (
```

Comment:
May want to sanitize date data so that we aren't expressing the difference in years.

![image](https://user-images.githubusercontent.com/2048050/39256500-10c71f4a-487d-11e8-8efc-69bf6b504f7f.png)

Reply:
I _think_ this can only occur for someone who resumes state from storage and doesn't do anything else. As soon as you re-run the report the `receivedAt` will get set and you'll have it from then on. 

As far as I can tell this is happening because it's getting `null` / `0` for `state.entities.receivedAt` and then running `distanceInWords(Date.now(), 0)` which calculates time between now and the unix epoch.

I don't think a guard for this is worth it since it goes away as soon as the user does anything other than just change sets in the tabs. Does that sound reasonable @tborres?